src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionList.java

```diff
@@ -238,7 +253,9 @@ public void onFailure(@NonNull Throwable t) {
     }
 
     @Restricted(DoNotUse.class)
-    @Terminator public static void saveAll() throws InterruptedException {
+    @SuppressWarnings("deprecation")
+    @Terminator(requires = EXECUTIONS_SUSPENDED, attains = LIST_SAVED)
```

Comment:
One thing I just thought about: if you have `workflow-api` installed but not `workflow-cps`, is the reactor smart enough to see that nothing attains `EXECUTIONS_SUSPENDED` and still allow this to run? Or does it block (bad), or get skipped (fine since you won't have any executions anyway)?

Reply:
> is the reactor smart enough to see that nothing attains `EXECUTIONS_SUSPENDED` and still allow this to run?

I wondered the same, but actually you can see from the fact that tests in this plugin pass that it is OK: it just considers `EXECUTIONS_SUSPENDED` to have attained immediately.

Reply:
> actually you can see from the fact that tests in this plugin pass that it is OK

Yeah I suspected that was the case, but wanted to make sure.